tree/ntuple/v7/src/RColumnElement.cxx

```diff
@@ -161,3 +165,29 @@ void ROOT::Experimental::Detail::RColumnElement<bool, ROOT::Experimental::EColum
       }
    }
 }
+
+void ROOT::Experimental::Detail::RColumnElement<float, ROOT::Experimental::EColumnType::kReal16>::Pack(
+   void *dst, void *src, std::size_t count) const
+{
+   float *floatArray = reinterpret_cast<float *>(src);
+   char *charArray = reinterpret_cast<char *>(dst);
```

Comment:
We can avoid the `memcpy` further down and directly write into the destination array:
```suggestion
   auto charArray = reinterpret_cast<std::uint16_t>(dst);
```